en/tutorials-and-examples/cms/articles-controller.rst

```diff
@@ -184,13 +184,12 @@ to be created. Start by creating an ``add()`` action in the
         {
             parent::initialize();
 
-            $this->loadComponent('Paginator');
-            $this->loadComponent('Flash'); // Include the FlashComponent
+            $this->set('articles', $this->paginate($this->Articles));
         }
 
         public function index()
         {
-            $articles = $this->Paginator->paginate($this->Articles->find());
+            $articles = $this->Articles->find();
```

Comment:
```suggestion
            $articles = $this->paginate($this->Articles);
```

Reply:
Paginate inside sub-controller

Reply:
Showing that you can pass both a Table or SelectQuery to paginate() is nice.